packages/ncids-css/packages/nci-autocomplete/internal/_nci-autocomplete.scss

```diff
@@ -45,7 +46,8 @@
 			background: none;
 		}
 
-		.minPlaceholderMsg {
+		//minPlaceholderMsg
+		.min-placeholder-message {
```

Comment:
Please add a note in the commit that you are changing the classname that affects javascript for autocomplete and that you've updated tests that go along with it